scipy/stats/_bootstrap.py

```diff
@@ -128,16 +153,17 @@ def _bootstrap_iv(data, statistic, axis, confidence_level, n_resamples,
 
     random_state = check_random_state(random_state)
 
-    return (data_iv, statistic, axis_int, confidence_level_float,
+    return (data_iv, statistic, vectorized, axis_int, confidence_level_float,
```

Comment:
Seems like you don't use this variable in the following. Are you returning it just to be consistent with the input values?

Reply:
The philosophy is that if I'm going to pass some variables into an input validation/standardization function and get some variables - possibly modified - back out, I don't want to have to think carefully about which to pass in and which to get back out. I think picking and choosing introduces the possibility in the future (e.g. if changes are made to how the IV function works) for subtle bugs that wouldn't have occurred had all the input validation/standardization code been inline. So I choose to pass everything in and return everything so it behaves as if the code were inline rather than a separate function.

Reply:
Ok thanks, just wanted to double check with you that it was your intent. Fine with me.